# Study 900 — Quality-Income — the teardown

The excess-of-cash Sharpe race, the quality-minus-yield HAC *t*, the paired block-bootstrap Sharpe-gap CI, the era cut, the vs-SPY races, the costed sleeves, and the 20-seed synthetic control.

In [1]:
R = {'start': '2013-11', 'end': '2026-06', 'n_months': 152, 'q_cagr': 11.0, 'q_vol': 13.9, 'q_sharpe': 0.697, 'q_maxdd': -22.4, 'q_wealth': 3.75, 'y_cagr': 10.13, 'y_vol': 13.7, 'y_sharpe': 0.65, 'y_maxdd': -27.5, 'y_wealth': 3.4, 'spy_cagr': 14.02, 'spy_vol': 14.5, 'spy_sharpe': 0.862, 'spy_maxdd': -23.9, 'spy_wealth': 5.27, 'gap': 0.047, 'diff_bps': 6.7, 'diff_ann': 0.81, 't_1s': 0.77, 't_nw': 0.57, 'ci_lo': -0.191, 'ci_hi': 0.241, 'p_neg': 0.355, 'era_e_gap': -0.007, 'era_e_diff': 0.6, 'era_e_t': 0.46, 'era_e_n': 74, 'era_l_gap': 0.06, 'era_l_diff': 1.0, 'era_l_t': 0.41, 'era_l_n': 78, 'qspy_gap': -0.165, 'qspy_diff': -2.81, 'qspy_t': -1.45, 'yspy_gap': -0.212, 'yspy_diff': -3.62, 'yspy_t': -1.44, 'cost3_q': 0.697, 'cost3_y': 0.65, 'cost10_q': 0.696, 'cost10_y': 0.65, 'drag3': 0.2, 'drag10': 0.7, 'turn': 0.6, 'null_mean_t': -0.4, 'null_sd_t': 1.03, 'null_fire': 1, 'planted_gap': 0.505, 'planted_t': 3.03, 'planted_diff': 6.59, 'cal_years': [2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025], 'cal_q': [-4.4, 27.4, 11.7, 27.7, -4.9, 6.3, 9.2, 5.6], 'cal_y': [-6.0, 22.2, -4.7, 25.5, 0.1, 3.9, 17.9, 9.3], 'cal_spy': [-4.6, 31.2, 18.3, 28.7, -18.2, 26.2, 24.9, 17.7]}

## The race — common window, excess of cash (minus BIL)

In [2]:
for name in ('q','y','spy'):
    lbl = {'q':'QUALITY (SCHD+NOBL)','y':'YIELD  (SPHD+VYM) ','spy':'SPY               '}[name]
    print(f"{lbl}: CAGR {R[name+'_cagr']:5.2f}%  vol {R[name+'_vol']:4.1f}%  "
          f"exSharpe {R[name+'_sharpe']:.3f}  maxDD {R[name+'_maxdd']:6.1f}%  "
          f"$1->{R[name+'_wealth']:.2f}")

QUALITY (SCHD+NOBL): CAGR 11.00%  vol 13.9%  exSharpe 0.697  maxDD  -22.4%  $1->3.75
YIELD  (SPHD+VYM) : CAGR 10.13%  vol 13.7%  exSharpe 0.650  maxDD  -27.5%  $1->3.40
SPY               : CAGR 14.02%  vol 14.5%  exSharpe 0.862  maxDD  -23.9%  $1->5.27


## Headline — quality vs yield Sharpe gap + HAC t on the monthly difference

The quality−yield monthly spread is cash-independent (cash cancels), so its mean / HAC *t* is the clean 'does quality out-earn yield?' statistic.

In [3]:
print(f"excess Sharpe : quality {R['q_sharpe']:.3f}  vs yield {R['y_sharpe']:.3f}  "
      f"-> GAP {R['gap']:+.3f}")
print(f"q-minus-y     : {R['diff_bps']:+.1f} bps/mo ({R['diff_ann']:+.2f}%/yr)  "
      f"one-sample t {R['t_1s']:+.2f}  NW t {R['t_nw']:+.2f}")
print(f"bootstrap gap : {R['gap']:+.3f}  95% CI [{R['ci_lo']:+.3f}, {R['ci_hi']:+.3f}]  "
      f"P(gap<0) = {R['p_neg']:.3f}")

excess Sharpe : quality 0.697  vs yield 0.650  -> GAP +0.047
q-minus-y     : +6.7 bps/mo (+0.81%/yr)  one-sample t +0.77  NW t +0.57
bootstrap gap : +0.047  95% CI [-0.191, +0.241]  P(gap<0) = 0.355


The Sharpe advantage is **positive in sign but insignificant**: the CI straddles zero and one block-draw in three has quality behind.

## Era cut (split 2020-01-01) — no stable magnitude

In [4]:
print(f"2013-11..2019-12 (n={R['era_e_n']}): gap {R['era_e_gap']:+.3f}  "
      f"diff {R['era_e_diff']:+.2f}%/yr  NW t {R['era_e_t']:+.2f}")
print(f"2020-01..2026-06 (n={R['era_l_n']}): gap {R['era_l_gap']:+.3f}  "
      f"diff {R['era_l_diff']:+.2f}%/yr  NW t {R['era_l_t']:+.2f}")

2013-11..2019-12 (n=74): gap -0.007  diff +0.60%/yr  NW t +0.46
2020-01..2026-06 (n=78): gap +0.060  diff +1.00%/yr  NW t +0.41


## Do the sleeves beat SPY? — both lag

In [5]:
print(f"quality vs SPY: gap {R['qspy_gap']:+.3f}  diff {R['qspy_diff']:+.2f}%/yr  NW t {R['qspy_t']:+.2f}")
print(f"yield   vs SPY: gap {R['yspy_gap']:+.3f}  diff {R['yspy_diff']:+.2f}%/yr  NW t {R['yspy_t']:+.2f}")

quality vs SPY: gap -0.165  diff -2.81%/yr  NW t -1.45
yield   vs SPY: gap -0.212  diff -3.62%/yr  NW t -1.44


## Costed — monthly rebalance turnover x one-way spread (long-only, no borrow)

Turnover is only the drift back to 50/50 (~0.6%/mo), so even a fat spread costs under 1 bp/yr — costs do not move the verdict.

In [6]:
print(f" 3 bps/side: quality {R['q_sharpe']:.3f}->{R['cost3_q']:.3f}  "
      f"yield {R['y_sharpe']:.3f}->{R['cost3_y']:.3f}  (drag {R['drag3']:.1f} bps/yr, turn {R['turn']:.1f}%/mo)")
print(f"10 bps/side: quality {R['q_sharpe']:.3f}->{R['cost10_q']:.3f}  "
      f"yield {R['y_sharpe']:.3f}->{R['cost10_y']:.3f}  (drag {R['drag10']:.1f} bps/yr)")

 3 bps/side: quality 0.697->0.697  yield 0.650->0.650  (drag 0.2 bps/yr, turn 0.6%/mo)
10 bps/side: quality 0.697->0.696  yield 0.650->0.650  (drag 0.7 bps/yr)


## Synthetic positive control — the machinery is unbiased

Live: the gap detector must NOT fire on the null and must recover a planted quality premium.

In [7]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
import numpy as np
from quality_income import data, strategy as st
null_t = np.array([st.synthetic_detect(data.synthetic_world(n_months=150, edge=0.0, seed=900+s))['t_nw'] for s in range(8)])
print(f"null (edge=0), 8 seeds: NW t mean {null_t.mean():+.2f} (sd {null_t.std(ddof=1):.2f}), |t|>=2 in {(abs(null_t)>=2).sum()}/8")
planted = st.synthetic_detect(data.synthetic_world(n_months=150, edge=0.03, seed=900))
print(f"planted (edge=+3%/yr): gap {planted['sharpe_gap']:+.3f}  NW t {planted['t_nw']:+.2f}  diff {planted['diff_ann_pct']:+.2f}%/yr")

null (edge=0), 8 seeds: NW t mean -0.07 (sd 1.07), |t|>=2 in 0/8
planted (edge=+3%/yr): gap +0.505  NW t +3.03  diff +6.59%/yr


## Verdict

- **Signal — Weak.** Quality edges yield by +0.9 pp/yr and a Sharpe gap of **+0.047**, with a genuinely shallower max drawdown (**-22.4%** vs **-27.5%**) from dodging the 2020 yield-sleeve crater — but the risk-adjusted edge is **not** significant (NW *t* = **+0.57**, bootstrap CI **[-0.19, +0.24]** straddles zero) and no era is significant. A real trap-avoidance *profile*, not a certified premium. The synthetic control recovers a *planted* edge cleanly (*t* = 3.03, fires on 1/20 nulls), so a real premium *would* have shown — this one didn't clear the bar. Short single-regime tape.
- **Tradability — Fragile.** The tilt is trivially buyable (cheap, liquid, long-only, monthly rebalance costs < 1 bp/yr — not a Mirage) — but there is no significant premium to bank; you buy a shallower-drawdown risk profile, not an edge. And both dividend sleeves trailed plain SPY by ~3-4 pp/yr.